routes/cart.js

```diff
@@ -1,8 +1,51 @@
-const express = require('express')
+const express = require('express');
+const req = require('express/lib/request');
```

Comment:
Not sure we need this? I think it's an automated import from one of the packages